src/app/layout/Page/index.tsx

```diff
@@ -7,13 +7,16 @@ import {
   HStack,
   IconButton,
   Stack,
+  useTheme,
 } from '@chakra-ui/react';
 import { FiArrowLeft, FiArrowRight } from 'react-icons/fi';
 
 import { useFocusMode } from '@/app/layout';
 import { useDarkMode } from '@/hooks/useDarkMode';
 import { useRtl } from '@/hooks/useRtl';
 
+import { useLayoutContext } from '../LayoutContext';
```

Comment:
Use `@/app/layout` for import (expose LayoutContext in layout/index.ts if needed)